_clients/javascript/getting-started.md

```diff
@@ -1,10 +1,11 @@
 ---
 layout: default
-title: JavaScript client
-nav_order: 100
+title: Getting started
+parent: JavaScript client
+nav_order: 1
 ---
 
-# JavaScript client
+# Getting started
```

Comment:
Getting Started (capitalized)

Reply:
Thank you for reviewing, @dblock! Capitalization is per our [style guide](https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md). I will make all other suggested changes.

Reply:
I have a new PR for this issue: #1358. I will close this PR and make the changes in the new one.